docs/sources/reference/components/local/local.file.md

```diff
@@ -71,4 +73,10 @@ local.file "secret_key" {
   filename  = "/var/secrets/password.txt"
   is_secret = true
 }
+
+# Then later:
+grafana_cloud.stack "receivers" {
+  stack_name = "mystack"
+  token = local.file.secret_key.content # <-- The file's contents are here
+}
```

Comment:
```suggestion
grafana_cloud.stack "receivers" {
  stack_name = "mystack"
  token = local.file.secret_key.content
}
```

I'd suggest we remove the in-line config file comments and add a better description of the Example up under the heading on line 69.  Maybe something like this? (I'm not sure I'm getting the description right here though)

``The following example shows a simple `local.file` configuration that watches a passwords text file and exports the content field.``